frontend/src/components/material/MaterialTable.vue

```diff
@@ -330,42 +337,32 @@ export default {
           id: key,
           quantity: mi.quantity,
           unit: mi.unit,
+          combinedQuantity: this.renderQuantity(mi),
           article: mi.article,
           listName: this.materialLists.find(
             (listItem) => listItem.value === mi.materialList
           ).text,
           new: true,
           serverError: mi.serverError,
           readonly: true,
-          class: 'new', // CSS class of new item rows
+          rowClass: 'new', // CSS class of new item rows
         })
       }
 
       return items
     },
-    materialItemsSorted() {
-      const items = this.materialItemCollection.items
-
-      // eager add new Items
-      for (const key in this.newMaterialItems) {
-        const mi = this.newMaterialItems[key]
-        items.push({
-          id: key,
-          quantity: mi.quantity,
-          unit: mi.unit,
-          article: mi.article,
-        })
-      }
-
-      return items.sort((a, b) => a.article.localeCompare(b.article))
-    },
   },
   methods: {
-    // remove existing item
-    deleteMaterialItem(materialItem) {
-      this.api
-        .del(materialItem.uri)
-        .catch((e) => this.$toast.error(errorToMultiLineToast(e)))
+    renderQuantity(item) {
+      if (item.quantity && item.unit) {
+        return `${item.quantity}\u00a0${item.unit}`
+      } else if (item.quantity) {
+        return `${item.quantity}\u00a0×`
```

Comment:
could you create a constant for this unicode character and give it a name.
i cannot imagine right away why it is here

Reply:
[f858d73](https://github.com/ecamp/ecamp3/pull/3797/commits/f858d73466aef010ba4d0e86cd53019feb09fdb9)